posts/inside-rust/2022-05-04-CTCFT-may.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: CTCFT 2022-05-16 Agenda
+author: Rust CTCFT Team
+---
+
+The next ["Cross Team Collaboration Fun Times" (CTCFT)][CTCFT] meeting will take
+place on Monday, 2022-05-16 at **11am US Eastern Time** ([click to see in your
+time zone][timezone]). You’ll find the full details (along with a calendar
+event, zoom details, etc) [on the CTCFT website][CTCFT-meeting].
+
+[CTCFT]: https://rust-lang.github.io/ctcft/
+[timezone]: https://everytimezone.com/s/6c2a0d08
+[CTCFT-meeting]: https://rust-lang.github.io/ctcft/meetings/2022-05-16.html
+
+## Agenda
+
+The theme for this month's CTCFT is **Embedded Rust**. We'll hear from some
+members of the Rust Embedded Working Group about the state of the ecosystem, as
+well as how async Rust is working for embedded systems. We also have some people
+coming in from the automotive industry to talk about how Rust use is starting to
+progress.
+
+- (5 min) Opening remarks 👋 ([angelonfira])
+- (15 min) A whirlwind tour of Embedded Rust ([jamesmunns])
+  - A brief history of the embedded-wg and use of Rust for embedded
+  - A look at how developing embedded rust looks like today
+  - A sample of patterns that are special to embedded rust, or differences from
```

Comment:
```suggestion
  - A look at how developing embedded Rust looks like today
  - A sample of patterns that are special to embedded Rust, or differences from
```

Reply:
I changed the filename so I can't commit this directly, but I did get it it :+1: thanks!